frontend/src/components/SignIn/SignIn.js

```diff
@@ -58,38 +58,43 @@ class SignIn extends Component {
               id="inputSignInEmail"
               className=" sign-in-input fullWidth form-control-lg"
               name="email"
+              pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
               required=""
               placeholder="Email"
-              onFocus={event => (event.target.setAttribute('placeholder', ''))}
-              onBlur={event => (event.target.setAttribute('placeholder', 'Email'))}
+              onFocus={event => event.target.setAttribute("placeholder", "")}
+              onBlur={event => event.target.setAttribute("placeholder", "Email")}
               onChange={this.handleChange}
             />
             <label htmlFor="inputSignInEmail">Email</label>
           </div>
 
           <div className="signInPassword form-group row">
             <input
-              type={isPasswordVisible ? 'text' : 'password'}
+              type={isPasswordVisible ? "text" : "password"}
               id="inputSignInPassword"
               className="sign-in-input fullWidth form-control-lg"
               name="password"
               required=""
               placeholder="Password"
-              onFocus={event => (event.target.setAttribute('placeholder', ''))}
-              onBlur={event => (event.target.setAttribute('placeholder', 'Password'))}
+              onFocus={event => event.target.setAttribute("placeholder", "")}
```

Comment:
Is it your linter that is making all the single quotes into double quotes? Other than that, everything looks good!